components/treasuryV2/Details/RealmAuthorityDetails/Config.tsx

```diff
@@ -248,12 +252,6 @@ export default function Config(props: Props) {
         )}
       </div>
       <NFTVotePluginSettingsDisplay className="mt-24" />
-      {editRealmOpen && (
-        <RealmConfigModal
```

Comment:
I think it would be best to delete this now unused modal from the codebase entirely. This will also tell you if it's used anywhere that you missed.